developers.diem.com/docs/core/run-local-network.md

```diff
@@ -5,7 +5,7 @@ title: Run a Local Network
 
 You can run a local test validator network to test and debug services you are developing for the Diem Blockchain and to build and test Move module code changes. This local test network is not part of the Diem Ecosystem and is only meant for testing and development purposes.
```

Comment:
While we're here, let's fix this run on sentence.

`You can run a local test validator network to test and develop against a Diem Blockchain, including Move changes.  This network is not part of the Diem Ecosystem and is only for testing and development purposes.